README.md

```diff
@@ -2,19 +2,99 @@
   <a href="https://github.com/actions/typescript-action/actions"><img alt="typescript-action status" src="https://github.com/actions/typescript-action/workflows/build-test/badge.svg"></a>
 </p>
 
-# Cherry-picking a pull request commit into branchs
+## Fork/Modified from this action: [cherry-pick-action](https://github.com/marketplace/actions/github-cherry-pick-action)
+
+# Xealth Auto Cherry Picker
 
 This action is designed to be used for cherry-pick commits from pull requests into release branches.
 
-GitHub Cherry Pick Action will:
+This action will:
 
 - Checkout triggered action.
 - Create new branch name `cherry-pick-${GITHUB_SHA}` from `branch` input.
 - Cherry-picking `${GITHUB_SHA}` into created `branch`
 - Push new `branch` to remote
 - Open pull request to `branch`
 
-## Example
+## Usage
+
+Simply add a `CP v(release branch)` label to your PR, and after merge to master: Github will open a cherry-pick label for you!
+
+An Example:
+ - Open a pull request for main on this repo: https://github.com/arivera-xealth/sample-repo
+ - Add the label `CP v1.0.0`
+ - On merge to `main`, a PR will be opened under the `cherry-pick` label
+ - That's it!
+
+You can see it in action here: https://github.com/arivera-xealth/sample-repo/pull/15
+
+----
+
+## Differences from [cherry-pick-action](https://github.com/marketplace/actions/github-cherry-pick-action)
+
+- The code has been reviewed and pulled into another repo for security reasons. 
```

Comment:
Omit? Just list differences.

Reply:
Yes, redoing the documentation as it is.